### PR TITLE
Standardize comment style. One space after //.

### PR DESCRIPTION
**Pull request by Yomi0**
I hope I didn't touch any code while I was doing this.

### CODE REVIEW

**Comment by marcelhollerbach** [comment]
Hi, sorry for the delay, I am on vacation. Looks fine, will merge it in in a few days. Will try to keep this formating :)

File: src/bin/fs_operation.c [hunk 1]
```diff
@@ -93,13 +93,13 @@ _operation_convert(Eina_List *files, const char *goal, void (*job)(Operation *op
...
-        //if there is a goal directory, build a not existing path
+        // if there is a goal directory, build a not existing path
         if (goal)
           {
-             //build the path
+             // build the path
              snprintf(path, sizeof(path), "%s/%s", goal, filename);
 
-             //check if the path allready exists
+             // check if the path allready exists
```
Comment (Yomi0) [inline src/bin/fs_operation.c]: allready to already...

File: src/lib/efm/fm_monitor.c [hunk 2]
```diff
@@ -108,7 +108,7 @@ _mon_err(void *data EINA_UNUSED, int type EINA_UNUSED, void *event)
 static Eina_Bool
 _mon_del(void *data EINA_UNUSED, int type EINA_UNUSED, void *event EINA_UNUSED)
 {
-   //we are doing nothing here ... a selfdel will result in a error call this are handeld there
+   // we are doing nothing here ... a selfdel will result in a error call this are handeld there
```
Comment (Yomi0) [inline src/lib/efm/fm_monitor.c]: handeld to handled

File: src/widgets/file/elm_file_icon2.c [hunk 3]
```diff
@@ -181,7 +181,7 @@ EOLIAN static void
 _elm_file_icon_evas_object_smart_del(Eo *obj, Elm_File_Icon_Data *pd EINA_UNUSED)
 {
    eo_do_super(obj, ELM_FILE_ICON_CLASS, evas_obj_smart_del());
-   //eio_monitor_del(pd->monitor);
+   // eio_monitor_del(pd->monitor);
```
Comment (Yomi0) [inline src/widgets/file/elm_file_icon2.c]: Ooh. I touched the code...

File: src/widgets/file/elm_file_display_preview.c [hunk 4]
```diff
@@ -41,14 +41,14 @@ filepreview_file_set(Evas_Object *w, Efm_File *file)
   const char *mime_type;
   Efm_File_Stat *st;
 
-  //char[] bytes = {"B", "KB", "MB", "GB", "TB", "PM", "EM", NULL}
+  // char[] bytes = {"B", "KB", "MB", "GB", "TB", "PM", "EM", NULL}
```
Comment (Yomi0) [inline src/widgets/file/elm_file_display_preview.c]: Again...

File: src/widgets/file/elm_file_display_helper.c [hunk 5]
```diff
@@ -118,7 +118,7 @@ sort_name_func(const void *data1, const void *data2)
         else
           return -1;
      }
-   else //if (config->sort.type == SORT_TYPE_EXTENSION)
+   else // if (config->sort.type == SORT_TYPE_EXTENSION)
```
Comment (Yomi0) [inline src/widgets/file/elm_file_display_helper.c]: Again.

File: src/widgets/file/elm_file_display_bookmark.c [hunk 6]
```diff
@@ -97,7 +97,7 @@ _setup_list(Evas_Object *o)
 
    if (!config) return;
 
-   //standart bookmarks
+   // standart bookmarks
```
Comment (Yomi0) [inline src/widgets/file/elm_file_display_bookmark.c]: Standart to standard

File: src/test/fileicon.c [hunk 7]
```diff
@@ -34,26 +34,26 @@ elm_main(int argc EINA_UNUSED, char **argv EINA_UNUSED)
    evas_object_show(tb);
 
    ic = eo_add(ELM_FILE_ICON_CLASS, win);
-   //eo_do(ic, efl_file_set("/home/marcel//filewithareallyreallylooooooooooooooooooooooooooooooooooooooonnnnggggggname.txt", NULL));
+   // eo_do(ic, efl_file_set("/home/marcel//filewithareallyreallylooooooooooooooooooooooooooooooooooooooonnnnggggggname.txt", NULL));
```
Comment (Yomi0) [inline src/test/fileicon.c]: again

Comment (marcelhollerbach) [inline src/test/fileicon.c]: commented out code, no problem :), same for the other code mods

**Pull request merged**